logging/google/cloud/logging/client.py

```diff
@@ -71,11 +71,23 @@ class Client(JSONClient):
     :param http: An optional HTTP object to make requests. If not passed, an
                  ``http`` object is created that is bound to the
                  ``credentials`` for the current object.
+
+    :type use_gax: bool or :class:`NoneType`
```

Comment:
Well the default value is `None` and `NoneType` is the type of `None` and this field describes the type. But its overkill to say `NoneType` every time there is an optional arg.